serveradmin/servershell/templates/servershell/choose_ip_addr.html

```diff
@@ -5,7 +5,19 @@
             <input type="text" class="form-control mb-2" id="ip_address" placeholder="Enter manually ...">
         </div>
         <div class="col-auto">
-            <button type="button" class="btn btn-success mb-2" onclick="submit_ip_address($('#ip_address').val());">Submit</button>
+            <button type="button" class="btn btn-success mb-2" onclick="submit_ip_address($('#ip_address').val());">
+                Submit
+            </button>
+        </div>
+    </div>
+    <div class="form-row align-items-center">
+        <div class="col-auto">
+            <input type="text" class="form-control mb-2" id="search_term" placeholder="Search ..."
+                   onchange="search($('#search_term').val());">
+        </div>
+        <div class="col-auto">
+            <button type="button" class="btn btn-success mb-2" onclick="search($('#search_term').val());">Search
```

Comment:
It would be nice to be able to reset the search either by a button or an empty value and display all values again.

![Screenshot from 2022-03-28 12-56-07](https://user-images.githubusercontent.com/3993306/160383799-5fe891e4-8434-4620-a0be-2a6e1d128be5.png)

Reply:
Fixed with empty value shows all items